rulebook.tex

```diff
@@ -1992,8 +1992,8 @@ \section{The Robotino System}
 may only occupy up to $25\%$ of this additional \SI{0.14}{\metre}
 (Robotino 2) or \SI{0.10}{\metre} (Robotino 3) wide ring around the
 robot. The only additional actuator allowed is one gripping device for
-workpieces which can be the original or a modified one. It also must
-be within the same diameter of the other additional hardware. The
+workpieces which can be the original or a modified one. In the rest position and while driving it also mustbe within the same diameter of the other additional hardware.
+During interaction with a machine, the gripper may be extended to twice the robot diameter, provided that the direction is within the angle of the two outermost points of the corresponting MPS. The
```

Comment:
This is not at all clear. If we allow to extend twice the diameter, that's 80 cm! I would think that this is way too far. Also, the what does "within the angle..." mean? Explain better, possibly also add a picture. I don't really understand what you mean here.

And please break lines at 80 characters.

Reply:
The new explanation is better, but I think it should be further clarified with a picture. I still thing twice the diameter is too much.

Reply:
The diameter of the Robotino is 45cm, the gripper should now look out another 45cm during operation. Such a definition (with Factor 2) is easier to control. But ok, we can also define not more than 35cm (width of the mps), so it can't look out on the other side. 
What do the others think?